R/caseduplicate.R

```diff
@@ -125,26 +142,29 @@ summary.duplicate <- function(object, dispersion = NULL, correlation = FALSE,
     if (!est.disp) {
       pvalue <- 2 * pnorm(-abs(tvalue))
       coef.table <- cbind(coef.p, s.err, tvalue, pvalue)
-      dimnames(coef.table) <- list(names(coef.p), c(dn,
-                                                    "z value", "Pr(>|z|)"))
+      dimnames(coef.table) <- list(
+        names(coef.p), c(dn, "z value", "Pr(>|z|)")
+      )
     }
     else if (df.r > 0) {
       pvalue <- 2 * pt(-abs(tvalue), df.r)
       coef.table <- cbind(coef.p, s.err, tvalue, pvalue)
-      dimnames(coef.table) <- list(names(coef.p), c(dn,
-                                                    "t value", "Pr(>|t|)"))
+      dimnames(coef.table) <- list(
+        names(coef.p), c(dn, "t value", "Pr(>|t|)")
+      )
     }
     else {
       coef.table <- cbind(coef.p, NaN, NaN, NaN)
-      dimnames(coef.table) <- list(names(coef.p), c(dn,
-                                                    "t value", "Pr(>|t|)"))
+      dimnames(coef.table) <- list(
+        names(coef.p), c(dn, "t value", "Pr(>|t|)")
+      )
     }
     df.f <- NCOL(Qr$qr)
   }
   else {
-    coef.table <- matrix(, 0L, 4L)
+    coef.table <- matrix(data = NA, 0L, 4L)
     dimnames(coef.table) <- list(NULL, c("Estimate", "Std. Error", "t value", "Pr(>|t|)"))
-    covmat.unscaled <- covmat <- matrix(, 0L, 0L)
+    covmat.unscaled <- covmat <- matrix(data = NA, 0L, 0L)
```

Comment:
Are you certain that `data = NA` is *exactly* the same as leaving this space blank? This is verbatim code from `stats::glm()`, which we know works. I see the aesthetics point, but why change a winning horse?